source/clear-linux/tutorials/hadoop.rst

```diff
@@ -0,0 +1,255 @@
+   said blocks. In this example, we are setting the number of block
+   replication to 1 from the default of 3 as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 6
+
+      <?xml version="1.0" encoding="UTF-8"?>
+      <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+      <configuration>
+      <property>
+      <name>dfs.replication</name>
+      <value>1</value>
+      </property>
+      <property>
+      <name>dfs.permission</name>
+      <value>false</value>
+      </property>
+      </configuration>
+
+#. Edit :file:`/etc/hadoop/mapred-site.xml`. This file configures all daemons
+   related to MapReduce: `JobTracker` and `TaskTrackers`. With MapReduce,
+   Hadoop can process big amounts of data in multiple systems. In our
+   example, we set :abbr:`YARN (Yet Another Resource Manager)` as our runtime
+   framework for executing MapReduce jobs as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 5,6
+
+      <?xml version="1.0" encoding="UTF-8"?>
+      <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+      <configuration>
+      <property>
+      <name>mapreduce.framework.name</name>
+      <value>yarn</value>
+      </property>
+      </configuration>
+
+#. Edit :file:`/etc/hadoop/yarn-site.xml`. This file configures all daemons
+   related to YARN: `ResourceManager` and `NodeManager`. In our example, we
+   implement the `mapreduce_shuffle` service, which is the default as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 4,5,8,9
+
+      <?xml version="1.0"?>
+      <configuration>
+      <property>
+      <name>yarn.nodemanager.aux-services</name>
+      <value>mapreduce_shuffle</value>
+      </property>
+      <property>
+      <name>yarn.nodemanager.auxservices.mapreduce.shuffle.class</name>
+      <value>org.apache.hadoop.mapred.ShuffleHandler</value>
+      </property>
+      </configuration>
+
+Configure your SSH key
+**********************
+
+#. Create a SSH key. If you already have one, just skip this step.
+
+   .. code-block:: bash
+
+      sudo ssh-keygen -t rsa
+
+
+#. Copy the key to your authorized keys.
+
+   .. code-block:: bash
+
+      sudo cat /root/.ssh/id_rsa.pub | sudo tee -a /root/.ssh/authorized_keys
+
+#. Log into the localhost. If no password prompt appears, you are ready to
+   run the Hadoop daemons.
+
+   .. code-block:: bash
+
+      sudo ssh localhost
+
+Run the Hadoop daemons
+**********************
+
+With all the configuration files properly edited, we are ready to start the
+daemons.
+
+When we format `NameNode`, it formats the meta-data related to data-nodes.
+Thus, all the information on the data nodes is lost and the nodes can be
+reused for new data.
+
+#. Format `NameNode` with the following command:
+
+   .. code-block:: bash
+
+      sudo hdfs namenode -format
+
+#. Start the DFS daemons `NameNode` and `DataNodes` with the following command:
+
+   .. code-block:: bash
+
+      sudo start-dfs.sh
+
+#. The console outputs:
```

Comment:
I suggest a more complete sentence is used here
The console output will be [similar to]: